iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Bus.java

```diff
@@ -240,4 +241,18 @@
      */
     void visitConnectedOrConnectableEquipments(TopologyVisitor visitor);
 
+    /**
+     * Get an iterable of the terminals connected to the bus.
+     */
+    default Iterable<? extends Terminal> getConnectedTerminals() {
+        return new LinkedList<>();
```

Comment:
Why a LinkedList and not an ArrayList?

Reply:
An old pre-Java 8 habit, as it is a constant empty list. Thus it does not require the whole ArrayList mechanism.

Reply:
replaced by `Collections.emptyList()` (and not the raw type `Collections.EMPTY_LIST`)